.gitignore

```diff
@@ -37,3 +37,7 @@ build_debug/
 
 #clangd cache dir
 .cache
+
+#patches, diffs, etc.
+*.patch
+*.diff
```

Comment:
this is a bit arbitrary — why the addition?

Reply:
I suppose that these kind of files are meant to be used locally for work in progress or for sending them to another dev.
I just noticed that they're recognized by a git in this repo so I automatically added them to the ignore list.
I think if there will ever be any patch or (less likely) diff to be tracked by version control, then it can be directly un-ignored here. What do you think?

Reply:
This would hide them from git status, making it easier for them to remain around. I actually do have a couple laying around, so it'd be impacted, plus if there ever was a need to ship a patch, one step less would be needed.

Reply:
Ok, I will remove it then ;)